---
string device_name
string serial_number
string firmware_version
string host_version
